package.json

```diff
@@ -68,6 +68,7 @@
         "onCommand:jupyter.selectNativeJupyterUriFromToolBar",
         "onCommand:jupyter.clearSavedJupyterUris",
         "onCommand:jupyter.filterKernels",
+        "onWalkthrough:jupyterWelcome",
```

Comment:
Something is off here. Did you start from the latest version of main? Meaning you did a 
- git checkout main
- git pull/fetch
- git checkout -b walkthrough

It looks like your baseline is from before your first change.

Reply:
If you do a 
- git fetch origin
- git rebase origin/main

That should fix the problem (after you fixup conflicts)

Reply:
No, I used the branch from my previous commit.... 🤦🏻‍♀️ will close this PR and submit a new one